starters/README.md

```diff
@@ -27,7 +27,7 @@
   <a href="https://npmcharts.com/compare/gatsby?minimal=true">
     <img src="https://img.shields.io/npm/dm/gatsby.svg" alt="Downloads per month on npm." />
   </a>
-  <a href="https://gatsbyjs.org/docs/how-to-submit-a-pr/">
+  <a href="https://gatsbyjs.org/contributing/how-to-contribute/">
```

Comment:
```suggestion
  <a href="https://gatsbyjs.org/contributing/how-to-open-a-pull-request/">
```
I think this would be the appropriate link

Reply:
no - on the home readme it is the other: 
https://github.com/gatsbyjs/gatsby/blob/master/README.md#L35-L37